EDU-17509 - Novo guia: CMS Portal (Legacy) overview (capa) (#2856)
vtexdocs/dev-portal-content: docs/guides/legacy-cms/cms-portal-legacy.md

---
title: "CMS Portal (Legacy)"
slug: "cms-portal-legacy"
hidden: false
createdAt: "2026-07-09T18:31:00.000Z"
excerpt: "VTEX's first storefront technology for building ecommerce websites with HTML, CSS, and JavaScript."
---

> CMS Portal (Legacy) is VTEX's first storefront technology that allows developers to build ecommerce websites using HTML, CSS, and JavaScript. Code management is done exclusively through the VTEX Admin, using the [Layout](https://help.vtex.com/docs/tracks/legacy-cms-portal) feature to create and manage templates, folders, and files.

Reviewer: ```suggestion
CMS Portal (Legacy) is VTEX's original storefront technology for building ecommerce websites using HTML, CSS, and JavaScript. Code management is done exclusively through the VTEX Admin, using the [Layout](https://help.vtex.com/docs/tracks/legacy-cms-portal) feature to create and manage templates, folders, and files.
```

> > ⚠️ CMS Portal (Legacy) is no longer available to newly created VTEX stores. If your store currently uses this technology, we strongly recommend [migrating to Store Framework](https://developers.vtex.com/docs/guides/vtex-io-documentation-migrating-storefront-from-legacy-to-io).

Reviewer: Sugestão: colocar este callout no início do texto.

CMS Portal (Legacy) relies on the following characteristics:

| Aspect | Description |
| --- | --- |
| Admin-based management | Code management is done exclusively through the VTEX Admin, without the need for local development tools or version control systems. |
| Template controls | Pre-built controls simplify the implementation of common storefront features like product displays, navigation menus, and search functionality. |
| Direct deployment | Content updates are made directly through the CMS without separate development or staging environments. |
| Layout customization | Customizations are managed by creating, organizing, and editing folders, files, and components within the Layout feature. |

## Technologies

To work with CMS Portal (Legacy), you should be familiar with the following technologies:

- [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML)
- [CSS](https://developer.mozilla.org/en-US/docs/Web/CSS)
- [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript)

## Quickstart

> CMS Portal (Legacy) allows you to create and manage your storefront entirely through the VTEX Admin. Follow these steps to begin working with it:

Reviewer: ```suggestion
For stores using CMS Portal (Legacy), the storefront is managed entirely through the VTEX Admin. Follow these steps to get started:
```

Reviewer: Só para não ficar contraditório com a ideia de que novas lojas não podem começar a usá-lo.
